samples/README.md

```diff
@@ -0,0 +1,4 @@
+Samples migrated
+================
+
+New location: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud_tasks/snippets
```

Comment:
We still have generated samples. Could you move this down to `samples/snippets/README.md`?